torch/quantization/_equalize.py

```diff
@@ -72,9 +75,17 @@ def cross_layer_equalization(module1, module2, output_axis=0, input_axis=1):
     weight1 = weight1 * inverse_scaling_factors
     weight2 = weight2 * scaling_factors
 
-    module1.weight = torch.nn.Parameter(weight1)
-    module1.bias = torch.nn.Parameter(bias)
-    module2.weight = torch.nn.Parameter(weight2)
+    if type(module1) in _supported_types:
+        module1.weight = torch.nn.Parameter(weight1)
+        module1.bias = torch.nn.Parameter(bias)
```

Comment:
these assignment can also be refactored to have a set_weight/set_bias function as well